cmd/tracee-ebpf/internal/printer/printer_test.go

```diff
@@ -0,0 +1,86 @@
+package printer_test
```

Comment:
1. package name should be lowercase. Don't use snake_case or camelCase in package names.
2. does this code compile? in the same folder you two different packages (!?)
3. the package name in this case should simply be `printer`

Reply:
1. This is what the vscode plugin suggested by default, I have no issue with changing it.
2. Yes, it compiles and is included in tests.
3. I had to separate it due to some circular dependency issue that would happen otherwise. The test code works as expected, I can change the package name if it's important.

Reply:
> yeah let's use `printer`

We can't though, that's my point, we need to use something else otherwise we get a circular dependency (this is because the printer config is parsed from the output flag).

If we want to keep the tests in the same package, I can move the config back to `flags` but I think it makes less sense than just keeping the test package separate.